### ISSUE TITLE
Build error when using a package with Angular Package Format

### ISSUE THREAD

**Issue by dxvladislavvolkov** [angular/angular-cli#16423]
# 🐞 Bug report

### Command (mark with an `x`)

- [ ] new
- [x] build
- [ ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc

### Is this a regression?

 Yes, the previous version in which this bug was not present was: ....

### Description

 

I am using ng7. I created and built my lib using ng-packagr.
When I import module from my lib in ng app I get an error when trying build application
This happens when I add an Input in my component, like this
```js
 @Input()
    get accessKey(): string {
        return this._getOption('accessKey');
    }
    set accessKey(value: string) {
        this._setOption('accessKey', value);
    }
```
## 🔬 Minimal Reproduction

You can reproduce it using the [repo](https://github.com/dxvladislavvolkov/app7)
Following the steps:

npm i
ng build --prod
and then you get the error

## 🔥 Exception or Error
<pre><code>

ERROR in ./node_modules/super-toasty/fesm5/super-toasty.js
Module build failed (from ./node_modules/@angular-devkit/build-optimizer/src/build-optimizer/webpack-loader.js):
TypeError: Cannot read property 'kind' of undefined
    at isAngularDecoratorMetadataExpression (C:\Users\volkov.vladislav\Downloads\downloadnew\universal\node_modules\@angular-devkit\build-optimizer\src\transforms\scrub-file.js:265:35)
    at checkNodeForDecorators (C:\Users\volkov.vladislav\Downloads\downloadnew\universal\node_modules\@angular-devkit\build-optimizer\src\transforms\scrub-file.js:77:21)
    at visitNodes (C:\Users\volkov.vladislav\Downloads\downloadnew\universal\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:16514:30)
    at Object.forEachChild (C:\Users\volkov.vladislav\Downloads\downloadnew\universal\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:16740:24)
    at checkNodeForDecorators (C:\Users\volkov.vladislav\Downloads\downloadnew\universal\node_modules\@angular-devkit\build-optimizer\src\transforms\scrub-file.js:68:31)
    at visitNode (C:\Users\volkov.vladislav\Downloads\downloadnew\universal\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:16505:24)
    at Object.forEachChild (C:\Users\volkov.vladislav\Downloads\downloadnew\universal\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:16635:21)
    at checkNodeForDecorators (C:\Users\volkov.vladislav\Downloads\downloadnew\universal\node_modules\@angular-devkit\build-optimizer\src\transforms\scrub-file.js:68:31)
    at visitNode (C:\Users\volkov.vladislav\Downloads\downloadnew\universal\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:16505:24)
    at Object.forEachChild (C:\Users\volkov.vladislav\Downloads\downloadnew\universal\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:16692:24)
    at checkNodeForDecorators (C:\Users\volkov.vladislav\Downloads\downloadnew\universal\node_modules\@angular-devkit\build-optimizer\src\transforms\scrub-file.js:68:31)
    at visitNode (C:\Users\volkov.vladislav\Downloads\downloadnew\universal\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:16505:24)
    at Object.forEachChild (C:\Users\volkov.vladislav\Downloads\downloadnew\universal\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:16703:24)
    at checkNodeForDecorators (C:\Users\volkov.vladislav\Downloads\downloadnew\universal\node_modules\@angular-devkit\build-optimizer\src\transforms\scrub-file.js:68:31)
    at visitNode (C:\Users\volkov.vladislav\Downloads\downloadnew\universal\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:16505:24)
    at Object.forEachChild (C:\Users\volkov.vladislav\Downloads\downloadnew\universal\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:16599:21)
</code></pre>

## 🌍 Your Environment
<pre><code>

</code></pre>

**Anything else relevant?**

**Comment by dxvladislavvolkov** [comment]
This has been fixed in the [8.3.7](https://github.com/angular/angular-cli/pull/15705). But I use angular 7 and to get the fix I need to update ng-cli and then ng to 8 versions.

**Comment by alan-agius4** [comment]
Hi, @dxvladislavvolkov, thanks for opening e issue and using Angular. As mentioned this was already fixed in Angular CLI v8.

Unfortunately such fixes are not back ported to LTS versions. As only security and critical fixes are.

Kindly update to the latest version. Thanks
